pennylane/transforms/__init__.py

```diff
@@ -224,3 +224,4 @@
 from .transpile import transpile
 from . import qcut
 from .qcut import cut_circuit, cut_circuit_mc
+from .unbroadcast_expand import unbroadcast_expand
```

Comment:
Don't forget to add it to the autosummary above so that it shows up in the docs :)